docs/style-guide/README.md

```diff
@@ -0,0 +1,7 @@
+# Ada Style and Quality Guide
+
+This book is converted from [WiKi Book](https://en.wikibooks.org/wiki/Ada_Style_Guide).
+
+Don't edit `*.mdx` files. 
+Use [converted](https://github.com/ada-lang-io/import-style-guide) instead.
```

Comment:
Can you remove this file and instead add a section to docs/getting-started/contribute.md?

Reply:
Maybe add some admonition to Ada_Style_Guide.mdx to mention it is a conversion from the wikibook style guide? @pyjarrett What's your opinion?

Reply:
Let's keep it as `_README.md`? As an extra notification to future authors.